qa-tests-backend/src/test/groovy/VulnScanWithGraphQLTest.groovy

```diff
@@ -172,7 +171,7 @@ class VulnScanWithGraphQLTest extends BaseSpecification {
     private GraphQLService.Response waitForImagesTobeFetched(String cveId , int retries = 30, int interval = 4) {
         Timer t = new Timer(retries, interval)
         while (t.IsValid()) {
-            def result2Ret = gqlService.Call(GET_IMAGE_INFO_FROM_VULN_QUERY, [id: cveId])
+            def result2Ret = gqlService.Call(GET_IMAGE_INFO_FROM_VULN_QUERY, [cve: cveId])
```

Comment:
There is no graphQL variable `cve`. Also, cve id in Postgres is `cve+os`. Refer to `postgres.IDFromPKs(...)`